spec/system/planning_applications/publicity/press_notice_spec.rb

```diff
@@ -530,6 +540,41 @@
         )
 
         expect(consultation.reload.end_date.to_date).to eq("Fri, 20 Oct 2023".to_date)
+
+        click_link "Edit publication"
+        fill_in "Optional comment", with: "Edited press notice comment"
+        click_button "Save"
+        click_link "Confirm press notice"
+        expect(page).to have_selector("p", text: "Comments: Edited press notice comment")
+
+        travel 1.hour
+        click_link "Add a new press notice response"
+        choose("Yes")
+        check("The application is for a Major Development")
+        click_button "Save and mark as complete"
+        perform_enqueued_jobs
+
+        click_link "Confirm press notice"
+        expect(page).to have_selector("p", text: "Date requested: #{PressNotice.last.requested_at.to_date.to_fs}")
+
+        # View past press notices
+        find("span", text: "View past press notices").click
+        within(".govuk-details__text") do
+          within "tbody tr:nth-child(1)" do
+            within("td:nth-child(1)") do
+              document = press_notice.reload.documents.first
+              expect(page).to have_content(document.name.to_s)
+              expect(page).to have_link("View in new window")
+            end
+
+            within("td:nth-child(2)") do
+              expect(page).to have_selector(".govuk-tag", text: "Published")
+              expect(page).to have_selector("p", text: "Date requested: #{press_notice.requested_at.to_date.to_fs}")
+              expect(page).to have_selector("p", text: "Date published: #{press_notice.reload.published_at.to_date.to_fs}")
+              expect(page).to have_selector("p", text: "Comments: Edited press notice comment")
+            end
+          end
```

Comment:
Using `nth-child` always feels a little fragile. Maybe adding classes to the rows and/or cells might be a bit clearer? `<td class="document-links">` and `<td class="document-details">` perhaps and then we can just check for the classes in the `within` blocks.

`tr:nth-child(1)` is probably more justified in this case as it's presumably important that it really is the first row and now some other.